.buildkite/pipeline.release.yml

```diff
@@ -0,0 +1,148 @@
+      # release" here, then we can use that to compute the final tag
+      # elsewhere in code. That may be less for people to juggle in
+      # their heads.
+
+  - label: ":github: Create Release Tag"
+    command:
+      .buildkite/scripts/git-tag
+    env:
+      BUILDKITE_CLEAN_CHECKOUT: true
+    plugins:
+      - grapl-security/vault-login#v0.1.2
+      - grapl-security/vault-env#v0.1.0:
+          secrets:
+            - GITHUB_TOKEN
+    agents:
+      queue: github-release
+
+  - wait
+
+  - group: "Build SDKs"
+    steps:
+      - label: ":python: Build Python SDK"
+        key: build-python-sdk
+        command:
+          - make python-sdk
+        plugins:
+          - improbable-eng/metahook#v0.4.1:
+              # We need to ensure we have the tag we just created
+              pre-command: git fetch --unshallow --tags
+          - docker#v3.11.0:
+              image: "${BUILD_ENVIRONMENT_IMAGE}"
+        artifact_paths:
+          - sdk/python/bin/dist/*.tar.gz
+
+      - label: ":nodejs: Build NodeJS SDK"
+        key: build-nodejs-sdk
+        command:
+          - make nodejs-sdk
+        plugins:
+          - improbable-eng/metahook#v0.4.1:
+              # We need to ensure we have the tag we just created
+              pre-command: git fetch --unshallow --tags
+          - docker#v3.11.0:
+              image: "${BUILD_ENVIRONMENT_IMAGE}"
+        artifact_paths:
+          - sdk/nodejs/bin/dist/*.tgz
+
+  - wait
+
+  # This will build the providers, package them up, upload them to
+  # Github, and announce them.
+  #
+  # We're intentionally publishing the SDKs after this; see below.
+  - label: ":github: Release!"
+    command:
+      # TODO: --skip-announce is only present for testing
+      # TODO: we really want this queue to be bigger so we can build more quickly
+      # TODO: I don't really love the fact that we're having to build these artifacts _in the release pipeline_.
+      - goreleaser release --skip-announce
+    plugins:
+      - grapl-security/vault-login#v0.1.2
+      - grapl-security/vault-env#v0.1.0:
+          secrets:
+            - GITHUB_TOKEN
+      - improbable-eng/metahook#v0.4.1:
+          pre-command: git fetch --unshallow
+      - docker#v3.11.0:
+          image: "${BUILD_ENVIRONMENT_IMAGE}"
+          environment:
+            - GITHUB_TOKEN
+    agents:
+      queue: github-release
+
+  - wait
+
+  # We're publishing the SDKs *after* the github release, because in
+  # order to *use* the SDKs, the binaries must be in the Github
+  # release first.
+  #
+  # This will hopefully save us from situations where we successfully
+  # publish the SDKs, but for whatever reason, the Github release
+  # fails.
+  - group: "Publish SDKs"
+    steps:
+      # TODO: Push to PyPI instead
+      - label: ":python: Publish Python SDK"
+        depends_on: build-python-sdk
+        command:
+          - .buildkite/scripts/twine-upload
+        plugins:
```

Comment:
AFAICT this pypi comment is already satisfied? twine-upload seems to do exactly that

Reply:
🤦 yes... will fix.